### PR TITLE
QA: Hold the browser session and page between scenarios

### PR DESCRIPTION
**Pull request by srbarrios**
## What does this PR change?

Hold the browser session (and page) between Cucumber scenarios, but not between Cucumber Features.

Bonus:
This also fixes a problem in XEN/KVM tests, that were opening a graphical console in full-screen mode in a new tab and not taking into account to close the tab, coming back to the previous page before end the scenario. 

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed

- [x] **DONE**

## Test coverage
- Cucumber tests were changed

- [x] **DONE**

## Links

Ports:
- Manager-4.0 https://github.com/SUSE/spacewalk/pull/14188
- Manager-4.1 https://github.com/SUSE/spacewalk/pull/14187

- [x] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Review by Bischoff — APPROVED** [review]
if it works, it's simply great !!!

File: testsuite/features/support/env.rb [hunk 1]
```diff
@@ -105,6 +105,16 @@ def enable_assertions
       debug_server_on_realtime_failure
     end
   end
+  page.instance_variable_set(:@touched, false)
+end
+
+# Reset the page if it's running the first scenario of a feature
+$first_scenario = true # This will be set every time we run a new feature
+Before do
+  if $first_scenario
```
Comment (Bischoff) [inline testsuite/features/support/env.rb:114]: or `next unless $first-scenario`, makes win one line :-)

**Review by Bischoff — APPROVED** [review]
lgtm, just left one comment

**Review by lkotek — APPROVED** [review]
LGTM :-) "G" stands for great in this case ;-)

**Comment by srbarrios** [comment]
@mcalmer @juliogonzalez we need that for a green in Head/Master. I will not be here on Monday, if you read that during the weekend please merge it, so you have a green Monday morning.

**Pull request merged**